### PR TITLE
fix(test): disable gsw and lower noise on a toy example

### PR DESCRIPTION
**Pull request by IceTDrinker**
### Resolves:

refs https://github.com/zama-ai/concrete-core-internal/issues/308
refs https://github.com/zama-ai/concrete-core-internal/issues/347
closes https://github.com/zama-ai/concrete-core-internal/issues/163

### Description

disable gsw tests as the corresponding functions are not used
to be re-enabled once we do something with them

### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] The PR description links to the related issue (to link an issue, use '#XXX'.)
* [ ] The tests on AWS have been launched and are successful (comment with @slab-ci cpu_test and/or @slab-ci gpu_test to trigger the tests)
* [ ] The draft release description has been updated
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

**Review by agnesLeroy — COMMENTED** [review]
Maybe we should remove the test code altogether instead of commenting it?

File: concrete-core/src/commons/crypto/gsw/tests.rs [hunk 1]
```diff
@@ -13,7 +13,7 @@ use concrete_commons::parameters::{
 use concrete_npe as npe;
 
 fn test_external_product_gsw<T: UnsignedTorus>() {
-    let n_tests = 10;
+    let n_tests = 1000;
```
Comment (aPere3) [inline concrete-core/src/commons/crypto/gsw/tests.rs:16]: Maybe we shouldnt change the number of tests here, if the purpose is just to deactivate some tests ?

**Review by aPere3 — APPROVED** [review]
Thanks for this PR ! 

I just have a minor comment on a number of samples. Otherwise looks good !

File: concrete-core/src/commons/crypto/gsw/tests.rs (hunk 1 above)
Comment (IceTDrinker) [reply concrete-core/src/commons/crypto/gsw/tests.rs:16]: yes I think I did some tests with more samples to have something working with stat test, I'll put it back to 10

Comment (IceTDrinker) [reply concrete-core/src/commons/crypto/gsw/tests.rs]: done ! I'll wait for the PR checks and then merge

**Pull request merged**